tests/security/security_tests.lst

```diff
@@ -36,4 +36,4 @@ tests/security/attributes/run_test.pl --scenario FullMsgSign_SubMsgSign_PayloadE
 tests/security/attributes/run_test.pl --scenario FullMsgSign_SubMsgEncrypt_PayloadEncrypt
 tests/security/attributes/run_test.pl --scenario FullMsgEncrypt_SubMsgSign_PayloadEncrypt
 tests/security/attributes/run_test.pl --scenario FullMsgEncrypt_SubMsgEncrypt_PayloadEncrypt
-tests/security/TopicWithoutPermissions/run_test.pl
+#tests/security/TopicWithoutPermissions/run_test.pl
```

Comment:
I think you meant to comment out the scenario that tested non-standard flag for fragmentation, not the TopicWithoutPermissions test?

Reply:
Yes, fixed now, thanks for catching.